applications/openshift/api-server/api_server_kubelet_client_cert/rule.yml

```diff
@@ -60,12 +62,10 @@ template:
   name: yamlfile_value
   vars:
     ocp_data: "true"
-    entity_check: "all"
     filepath: {{{ openshift_filtered_path(default_api_path, default_jqfilter) }}}
-    yamlpath: '.apiServerArguments["kubelet-client-certificate"][:]'
-    xccdf_variable: var_apiserver_kubelet_client_cert
-    embedded_data: "true"
+    yamlpath: "[:]"
+    check_existence: "at_least_one_exists"
+    entity_check: "all"
     values:
-    - value: '(.+)'
-      type: "string"
-      operation: "pattern match"
+      - value: "(.*?)"
```

Comment:
it will work because we will output '/etc/kubernetes/certs/kubelet/tls.crt' only if there is a match, and we have a `check_existence: "at_least_one_exists"`.

Reply:
thanks for the explanation!